account_payment_check_number/README.rst

```diff
@@ -0,0 +1,20 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+============================
+Account Payment Check Number
+============================
+
+This module will manage check number on supplier payment screen.
```

Comment:
```suggestion
This module makes the check number field visible in the Payments list.
```

Reply:
Usage: "On the Payments list, the Check Number field is available as an option, using the far right icon in the list header."